README.md

```diff
@@ -1356,6 +1356,57 @@ $ ssh-add -E md5 -l
 
 When using the key `pinentry` will be invoked to request the key's passphrase. The passphrase will be cached for up to 10 minutes idle time between uses, to a maximum of 2 hours.
 
+## Remote Machines (agent forwarding)
+
+If you want to use your YubiKey to sign a git commit on a remote machine, or ssh through another layer, then this is possible using "Agent Forwarding".  To do this, you need to already have shell access to your remote machime, and your YubiKey setup on your host machine.
```

Comment:
"Agent Forwarding" is not a completely correct term, as it's responsible only for SSH authentication.
Where the "RemoteForward" ssh feature helps to do the GPG trick.

Reply:
I puzzled over that too; would help to clarify that ssh-agent forwarding isn't the same as gpg-agent forwarding.  But while `~/.ssh/config` refers to the former as `ForwardAgent` and the latter as `RemoteForward`, I think the generic term "Agent Forwarding" is used for both, e.g. as it is by the GPG wiki: https://wiki.gnupg.org/AgentForwarding.